sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/CosmosAzureEnvironments.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.kafka.connect.implementation;
+
+public enum CosmosAzureEnvironments {
+    AZURE("azure"),
+    AZURE_CHINA("AzureChina"),
+    AZURE_US_GOVERNMENT("AzureUsGovernment"),
+    AZURE_GERMANY("AzureGermany");
```

Comment:
I am curious to know where did you get this list?
Will it extend?

Reply:
I copied from the managementSDK -> they are unlikely change, but still possible I guess. I did not start with adding management SDK into the dependency as it will bring in a lot other dependencies. So added a TODO comment so to discuss further about what to do before the release

Reply:
Discussed offline, will use validator pattern